### PR TITLE
fix cache size in warm up

### PR DESCRIPTION
**Pull request by almogdepaz**
## Motivation
wrong cache size, the size is set with layers in mind but the cap method of the cache refers to number of blocks

## Changes
size of cache is determined by layers not blocks 

## Test Plan
same

### CODE REVIEW

File: cmd/node/node.go [hunk 1]
```diff
@@ -511,7 +511,7 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId,
 	if mdb.PersistentData() {
 		trtl = tortoise.NewRecoveredTortoise(mdb, app.addLogger(TrtlLogger, lg))
 		msh = mesh.NewRecoveredMesh(mdb, atxdb, app.Config.REWARD, trtl, app.txPool, atxpool, processor, app.addLogger(MeshLogger, lg))
-		go msh.CacheWarmUp()
+		go msh.CacheWarmUp(app.Config.LayerAvgSize)
```
Comment (y0sher) [inline cmd/node/node.go:506]: so we don't want to wait for the cache to be warmed up ?

**Review by y0sher — APPROVED** [review]

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Comment by noamnelke** [comment]
Merged as part of **Consolidated linting (#1958)**

**Pull request closed without merging**